dashboard/app/models/peer_review.rb

```diff
@@ -142,9 +142,25 @@ def self.create_for_submission(user_level, level_source_id)
     end
   end
 
+  def self.get_review_completion_status(user, script)
+    if user &&
+        script.peer_reviews_to_complete.try(:>, 0) &&
+        Plc::EnrollmentUnitAssignment.exists?(user: user, plc_course_unit: script.plc_course_unit)
+      reviews_done = PeerReview.where(reviewer: user, script: script, status: [0, 1, 2]).size
+
+      if reviews_done >= script.peer_reviews_to_complete
+        Plc::EnrollmentModuleAssignment::COMPLETED
+      elsif reviews_done > 0
+        Plc::EnrollmentModuleAssignment::IN_PROGRESS
+      else
+        Plc::EnrollmentModuleAssignment::NOT_STARTED
+      end
+    end
+  end
+
   def self.get_peer_review_summaries(user, script)
     if user &&
-        script.professional_learning_course? &&
+        script.peer_reviews_to_complete.try(:>, 0) &&
```

Comment:
This shows up twice, can we extract it to a more friendly method on the `Script` model?

E.g. `Script#has_peer_reviews_to_complete?`